scripts/build-chms-history.php

```diff
@@ -1,17 +1,15 @@
 <?php
 	/**
-	 * +----------------------------------------------------------------------+
-	 * | PHP Version 5                                                        |
-	 * +----------------------------------------------------------------------+
-	 * | Copyright (c) 1997-2013 The PHP Group                                |
-	 * +----------------------------------------------------------------------+
-	 * | This source file is subject to version 3.01 of the PHP license,      |
-	 * | that is bundled with this package in the file LICENSE, and is        |
-	 * | available through the world-wide-web at the following url:           |
-	 * | http://www.php.net/license/3_01.txt.                                 |
-	 * | If you did not receive a copy of the PHP license and are unable to   |
-	 * | obtain it through the world-wide-web, please send a note to          |
-	 * | license@php.net so we can mail you a copy immediately.               |
+     * +----------------------------------------------------------------------+
+     * | Copyright (c) 1997-2023 The PHP Group                                |
+     * +----------------------------------------------------------------------+
+     * | This source file is subject to version 3.01 of the PHP license,      |
+     * | that is bundled with this package in the file LICENSE, and is        |
+     * | available through the world-wide-web at the following url:           |
+     * | https://www.php.net/license/3_01.txt.                                |
+     * | If you did not receive a copy of the PHP license and are unable to   |
+     * | obtain it through the world-wide-web, please send a note to          |
+     * | license@php.net, so we can mail you a copy immediately.              |
```

Comment:
Indented with spaces here, but tabs are used on the lines before and after this.